conflux/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "conflux-network-adapter",
+  "version": "0.1.0",
+  "main": "index.js",
+  "repository": "https://github.com/Conflux-Network-Global/conflux-network-adapter.git",
+  "author": "Jonas Hals <jonas@smartcontract.com> (modified by Aaron Lu <aaron@conflux-chain.org>)",
+  "license": "MIT",
+  "dependencies": {
+    "@chainlink/ea-bootstrap": "^0.1.3",
+    "@chainlink/external-adapter": "^0.2.6",
+    "dotenv": "^8.2.0",
+    "js-conflux-sdk": "^1.5.10",
+    "web3-utils": "^1.3.3"
+  },
+  "devDependencies": {
+    "eslint": "^7.14.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^5.0.0"
+  },
+  "scripts": {
+    "lint": "yarn eslint ."
```

Comment:
Take as reference any other adapter, every `package.json` should be pretty similar